gradle/dependencies.gradle

```diff
@@ -23,7 +23,7 @@ ext {
 
 versions.robovm = "2.3.14"
 versions.gwt = "2.8.2"
-versions.gwtPlugin = "1.0.9"
+versions.gwtPlugin = "1.1.16"
 versions.gretty = "3.0.6"
```

Comment:
Gretty is on version 3.0.7 now, at least. 3.0.6 does still work, but I think 3.0.7 will be the last update to the Gretty 3.x line for a while. This is important because Gretty 4.0.0 (at least from what I recall) needs Java 11 to run, so we can't go up to the 4.x line.

Reply:
Not aware of any issues with 3.0.7, the only reason I've kept 3.0.6 is it was already being used by `gdx-tests-gwt`and I don't do any html so I wanted to touch as little as possible just to make it run. Should be fixed now.